docs/source/development/developer_setup.md

```diff
@@ -121,20 +121,20 @@ may have parallel `venv` for different Python versions and you can check this us
 `poetry env list`:
 
 ```bash
-dorme@MacBook-Pro virtual_rainforest % poetry env list
+dorme@MacBook-Pro virtual_ecosystem % poetry env list
 virtual-rainforest-Laomc1u4-py3.10 (Activated)
 virtual-rainforest-Laomc1u4-py3.9
-dorme@MacBook-Pro virtual_rainforest %
+dorme@MacBook-Pro virtual_ecosystem %
 ```
 
 You can now launch a shell using that `venv`.
 
 ```bash
-dorme@MacBook-Pro virtual_rainforest % poetry shell
+dorme@MacBook-Pro virtual_ecosystem % poetry shell
 Spawning shell within /Users/dorme/Library/Caches/pypoetry/virtualenvs/virtual-rainforest-Laomc1u4-py3.10
-dorme@MacBook-Pro virtual_rainforest % . /Users/dorme/Library/Caches/pypoetry/virtualenvs/virtual-rainfor
+dorme@MacBook-Pro virtual_ecosystem % . /Users/dorme/Library/Caches/pypoetry/virtualenvs/virtual-rainfor
```

Comment:
I guess this is not urgent but the name of the virtual environment might change in the future

Reply:
Hah. I had - but hadn't saved the file.